ludwig/features/image_feature.py

```diff
@@ -313,13 +335,16 @@ def add_feature_data(
 
                 proc_df[feature[PROC_COLUMN]] = backend.df_engine.map_objects(
                     input_df[feature[COLUMN]],
-                    lambda file_path: read_image_and_resize(get_abs_path(src_path, file_path))
+                    lambda img_store: read_image_and_resize(
+                        get_abs_path(src_path, img_store))
+                    if isinstance(img_store, str) else img_store
                 )
         else:
             backend.check_lazy_load_supported(feature)
 
-            all_file_paths = [get_abs_path(src_path, file_path)
-                              for file_path in input_df[feature[NAME]]]
+            all_img_stores = [get_abs_path(src_path, img_store)
+                              if isinstance(img_store, str) else img_store
+                              for img_store in input_df[feature[NAME]]]
```

Comment:
Also here, should likely be COLUMN not NAME

Reply:
Done